src/main/java/com/exasol/containers/ssh/DirectorySelector.java

```diff
@@ -0,0 +1,63 @@
+package com.exasol.containers.ssh;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * Considers a number of candidates for a directory with respect to specific conditions for each candidate.
+ */
+public class DirectorySelector {
+
+    private final Path parent;
+
+    private boolean found = false;
+    private Path path;
+    private boolean knownAsExisting = false;
+
+    public DirectorySelector() {
+        this(Path.of(""));
```

Comment:
```suggestion
        this(Path.of("."));
```
This would make it clear that this is the current working dir.